scavenger_hunt_next_25/linux/CMakeLists.txt

```diff
@@ -0,0 +1,128 @@
+# Project-level configuration.
```

Comment:
I didn't think `flutterfire` supported Linux?

Reply:
Working on amending it!